README.md

```diff
@@ -17,13 +17,13 @@ Bedrock also provides automation for deploying Kubernetes clusters with Terrafor
 
 A Bedrock deployment follows three general steps at a high level:
 
-1. Define a [Fabrikate](https://github.com/Microsoft/fabrikate) definition for your deployment.
-2. [Deploy a CI/CD pipeline](./gitops) to build resource manifests from this deployment definition.
-3. [Create and deploy](./cluster) a Kubernetes environment with Flux.
+1. [Create and deploy azure-simple](./docs/azure-simple/README.md) a Kubernetes environment that uses Flux.
+2. Define a [Fabrikate](./docs/fabrikate/README.md) definition.
+3. [Deploy a CI/CD pipeline](./docs/devops/README.md) to build resource manifests for this deployment. 
 
 Our cluster creation templates include deployments of a [cloud-native stack](https://github.com/timfpark/fabrikate-cloud-native) by default.  This is intended to be replaced with your own resource manifest repo, but you can also take advantage of this by jumping directly to step #3 if you'd like to give Bedrock a try before defining your own deployment.
 
-The easiest way to try Bedrock is to start with our [azure-simple](https://github.com/Microsoft/bedrock/tree/master/cluster/environments/azure-simple) deployment template or with [minikube](https://github.com/Microsoft/bedrock/tree/master/cluster/environments/minikube) to try it locally.   
+The easiest way to try Bedrock is to start with our [azure-simple](./docs/azure-simple/README.md) deployment. 
```

Comment:
The terminology we use elsewhere is to call this not a deployment but an “environment template” - let’s snap to that here.